l10n_it_fatturapa/views/partner_view.xml

```diff
@@ -8,18 +8,19 @@
         <field name="arch" type="xml">
             <notebook position="inside">
             <page name="fatturapa" string="Electronic Invoice" groups="account.group_account_invoice">
-                <group name="fatturapa_group">
-                    <label for="electronic_invoice_subjected"/>
-                    <field name="electronic_invoice_subjected"/>
-
-                    <group attrs="{'invisible': [('electronic_invoice_subjected', '=', False)]}">
+	        <group name="fatturapa_group">
+                    <group>
+                        <field name="electronic_invoice_subjected"/>
+                        <field name="electronic_invoice_obliged_subject" attrs="{'invisible': [('electronic_invoice_subjected', '=', False)]}"/>
```

Comment:
Bisognerebbe aggiungere un'onchange su `electronic_invoice_obliged_subject` che, nel momento in cui l'utente lo deseleziona, il sistema dovrebbe riportare i campi invisibili nello stato originario.
Altrimenti uno potrebbe impostare un codice destinatario e deselezionare `electronic_invoice_obliged_subject`; il sistema memorizzerebbe e poi utilizzerebbe un codice destinatario probabilmente sbagliato.